public/index.html

```diff
@@ -21,36 +18,6 @@
       ga('create', 'UA-36512724-2', 'auto');
       ga('send', 'pageview');
     </script>
-
-    <style>
-
-      html {
-        box-sizing: border-box; /* border-box */
-      }
-
-      *,
-      *:before,
-      *:after {
-        box-sizing: inherit;
-      }
-
-      body {
-        margin: 0; /* ma0 */
-        background: #1b1f24; /* navy */
-        color: #fff; /* white */
-        min-height: 100vh; /* min-vh-100 */
-      }
-
-      a {
-        color: #009dcf; /* blue */
-      }
-
-      code {
-        color: #42c2b3; /* turquoise */
-      }
-
-    </style>
```

Comment:
This style reset is in tachyons-egghead now so you get it when you `import 'tachyons-egghead'` in a project; Once egghead-rails updates to the styleguide, it can `yarn upgrade tachyons-egghead egghead-ui` and then remove these types of resets now as well; here are a couple I see right off the bat with a quick peek in the project.

- https://github.com/eggheadio/egghead-rails/blob/master/client/app/assets/styles/globals/base.scss#L10
- https://github.com/eggheadio/egghead-rails/blob/master/client/package.json#L58
- https://github.com/eggheadio/egghead-rails/blob/master/client/package.json#L37 and use latest `<Request>` with a `<WrappedRequest>` like egghead-instructor-center: https://github.com/eggheadio/egghead-instructor-center/tree/master/src/components/WrappedRequest
- https://github.com/eggheadio/egghead-rails/blob/master/client/package.json#L107 once using latest `<Request>`

There might be more. Anyways, a few ideas for consolidating duplicate efforts :)